Eludia/Presentation.pm

```diff
@@ -1939,8 +1939,7 @@ sub get_composite_table_headers {
 
 		push @$result_headers, $headers -> [$i] -> [$j];
 
-		if ($headers -> [$i] -> [$j] -> {colspan}) {
-
+		if ($headers -> [$i] -> [$j] -> {colspan} || $_REQUEST {xls} && @{$headers -> [$i + 1]} > 0) {

```

Comment:
$headers->[$i + 1] will be created as empty if it does not already exist
What will guarantee that it exists?